### DISCUSSION TITLE
TuringCourses/math_phys/physics1/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#169] [General]
# TuringCourses/math_phys/physics1/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/physics1/

**Comment by 626lt** [comment]
刘钊老师中文授课，英文教材ppt。刘钊老师讲课清晰有条理，远胜于某些计院老师，而且答疑耐心细致。人很nice，上课挺有趣的，要是普物能听懂就更好了。刘钊老师跟外教班、路欣老师是一同考试，教材作业一样。个人认为课程难度很大，需要额外投入时间精力来学习理解。如果想要取得一个相对满意的成绩，平时作业一定要认真写，今年的作业中就包含了去年的期末考题目。因为是外文教材，所以很难找到适配的练习题，所以一定一定要好好写作业。除此以外，上课认真听也不一定能听懂，所以习题quiz都要认真对待。单就考试来说，吃透习题加上认真复习以及**听从老师的强调**就可以取得相对可以的成绩，老师会捞的（确信）。22级的期末考试是10个填空题加上几个大题的形式，期末内容也主要是在后半段的热学领域。
